markdown/3.0/en/documentation/include-scripts.md

```diff
@@ -2,45 +2,23 @@
 
 ---
 
-The FACT-Finder Web Components build contains three files:
-1. `elements.build_with_dependencies.html` 
-    * This file contains all HTML pieces needed to provide DOM pieces (if needed) and a reference to `elements.build.js`
-2. `elements.build.js`
-    * Contains all the JavaScript code to make FACT-Finder Web Components work
-3. `webcomponents-lite.min.js`
-    * Contains the required polyfill to have FACT-Finder Web Components technology working in older browsers which don't support the Web Components spec natively. This is **always** required for all versions <= 1.3 
-
+The following section shows a boilerplate for including FACT-Finder Web Components in your website. `custom-elements-es5-adapter.js` and `webcomponents-loader.js` will load the required Polyfills minimized to the needs of the calling browser. 
```

Comment:
The following section shows the boilerplate for integrating FACT-Finder Web Components into your website. `custom-elements-es5-adapter.js` and `webcomponents-loader.js` will load the required Polyfills optimized to the needs of the calling browser.